src/errors.ts

```diff
@@ -34,6 +38,18 @@ class ConnectionError extends WidgetError {
   }
 }
 
+export class SwapError extends WidgetError {
+  constructor(config: WidgetErrorConfig) {
+    super({
+      ...config,
+      header: config.header ?? DEFAULT_DISMISSABLE_ERROR_HEADER,
```

Comment:
This is never used - all of your `SwapError`'s constructors pass in `Swap Error`.
Could you rm `DEFAULT_DISMISSABLE_ERROR_HEADER`, and inline

    `t`Swap Error`

here instead?

Reply:
Alternatively, define a `DismissableWidgetError` that takes on these defaults, and then derive `SwapError` from that.